src/Tools/CoreFx.Tools/CoreFx.Tools.builds

```diff
@@ -5,7 +5,6 @@
     <Project Include="$(MSBuildThisFileDirectory)$(MSBuildProjectName).csproj" />
     <Project Include="$(MSBuildThisFileDirectory)$(MSBuildProjectName).csproj">
```

Comment:
Given the way we build this https://github.com/dotnet/corefx/blob/master/build.proj#L41, this entire file can probably be deleted.